src/main/java/com/mosaic/Apiclienstub.java

```diff
@@ -0,0 +1,8 @@
+package com.mosaic;
+
+public class Apiclienstub {
```

Comment:
Looks great!

Just a tiny remark on naming conventions in Java:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Better to follow these. There is no logic behind really. It is just an agreement in Java world